### PR TITLE
Should check the value range of BigIntColumn

### PR DESCRIPTION
**Pull request by brfrn169**
This PR adds the check of the value range of `BigIntColumn`. Please take a look!

### CODE REVIEW

**Review by feeblefakie — DISMISSED** [review]
LGTM! Thank you!
(we just forgot to check?)

**Comment by brfrn169** [comment]
> LGTM! Thank you! (we just forgot to check?)

Yes, we just forgot to check.

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank you!

---

I got expected error correctly when I tried to insert `9007199254740993` to the `BIGINT` column.

* Test code (excerpt)
  ```java
          try {
              Put insert = Put.newBuilder()
                  .namespace("s0")
                  .table("t0")
                  .partitionKey(Key.ofInt("c1", 1))
                  .bigIntValue("c2", 9007199254740993L)
                  .build();
              tx0.put(insert);
              tx0.commit();
          } catch (Exception e) {
              System.out.println(e.getMessage());
              e.printStackTrace();
              tx0.abort();
          }
  ```
* Error (expected error)
  ```java
  Out of range column value for BigInt
  java.lang.IllegalArgumentException: Out of range column value for BigInt
          at com.google.common.base.Preconditions.checkArgument(Preconditions.java:145)
          at com.scalar.db.io.BigIntColumn.<init>(BigIntColumn.java:36)
          at com.scalar.db.io.BigIntColumn.<init>(BigIntColumn.java:27)
          at com.scalar.db.io.BigIntColumn.of(BigIntColumn.java:133)
          at com.scalar.db.api.PutBuilder$Buildable.bigIntValue(PutBuilder.java:127)
          at com.example.InsertBigintValue.main(InsertBigintValue.java:65)
  ```

File: core/src/main/java/com/scalar/db/io/BigIntColumn.java [hunk 1]
```diff
@@ -31,6 +33,7 @@ private BigIntColumn(String name) {
 
   private BigIntColumn(String name, long value, boolean hasNullValue) {
     this.name = Objects.requireNonNull(name);
+    checkArgument(value >= MIN_VALUE && value <= MAX_VALUE, "Out of range column value for BigInt");
```
Comment (komamitsu) [inline core/src/main/java/com/scalar/db/io/BigIntColumn.java:36]: Maybe adding the actual `value` in the error message would help users?

Comment (brfrn169) [reply core/src/main/java/com/scalar/db/io/BigIntColumn.java]: Thank you for your feedback! I added the actual `value` in the error message. Please take a look again when you have time!

**Review by komamitsu — APPROVED** [review]
LGTM. Thanks!

**Comment by brfrn169** [comment]
Merging this for now.

@Torch3333 Can you please take a look when you come back?

**Review by Torch3333 — COMMENTED** [review]
LGTM, thank you!

**Pull request merged**